import io.netty.resolver.dns.DnsNameResolverException;
public class FunctionUse9412 {
public void funcUse() {
DnsNameResolverException dnsnameresolverexception = new DnsNameResolverException();
dnsnameresolverexception.validateQuestion(null);
}
}